### ISSUE TITLE
[New group] China, ChengDu

### ISSUE THREAD

**Issue by TechQuery** [strapi/strapi-meetups#9]
## Background Information

- **Name**: Yao Shi
- **Company:** [freeCodeCamp ChengDu community](https://fcc-cd.dev/)
- **Country, City:** China, ChengDu
- **Personal Website or blog:** https://tech-query.me/
- **Twitter:** @TechQuery_FCC

## Contributions to Strapi

- [Quick start in Chinese](https://tech-query.me/development/headless-cms-strapi/)
- Strapi awesomes
  - [CKEditor plugin](https://github.com/TechQuery/strapi-plugin-ckeditor)
  - [OrgServer scaffold](https://github.com/kaiyuanshe/OrgServer)
  - [MobX SDK](https://github.com/EasyWebApp/MobX-Strapi)

## Strapi support needed

- [ ] Finding speakers
- [ ] Finding host venues
- [x] Finding sponsors
- [x] Promoting my events
- [x] I only need Strapi swag

## Community Guidelines

- [x] I have read & agree to respect Strapi's [code of conduct](https://github.com/strapi/strapi/blob/master/CODE_OF_CONDUCT.md) and [community guidelines](https://github.com/strapi/strapi-meetups/blob/master/guidelines.md)
- [x] I have checked that there is no existing Strapi user group in this city.

**Comment by vcoisne** [comment]
hello @TechQuery sorry about the (very) late reply. We think now might be a good time to bring back this meetup program to life. Would you still be interested in launching a Strapi user group in ChengDu ?

**Comment by TechQuery** [comment]
> hello @TechQuery sorry about the (very) late reply. We think now might be a good time to bring back this meetup program to life. Would you still be interested in launching a Strapi user group in ChengDu ?

@vcoisne Yes, I have prepared Strapi meetups for almost 3 years, and rewrite [the MobX SDK for Strapi v4](https://github.com/idea2app/MobX-RESTful/tree/main/wrapper/Strapi).

Just tell me how can I cooperate with you guys, no more 9 months waiting in LinkedIn message.

**Comment by vcoisne** [comment]
Thanks @TechQuery for the quick reply. I answered your DM. Looking forward to our chat :)
